src/data/pipeline-builds/all-pipeline-builds-query.ts

```diff
@@ -0,0 +1,44 @@
+import { apiPaths, createApiPath } from "../api";
+import { UseQueryOptions, useQuery } from "@tanstack/react-query";
+import { FetchError } from "@/lib/fetch-error";
+import { PipelineBuildPage } from "@/types/pipeline-builds";
+export type PipelineBuildOverview = {
+	buildId: string;
+};
+
+export function getPipelineBuildQueryKey({ buildId }: PipelineBuildOverview) {
+	return ["builds", buildId];
```

Comment:
to keep it consistent, we should put the actual path segments in that array, which means `pipeline-builds` in this case.